packages/uui-tabs/lib/uui-tab-group.element.ts

```diff
@@ -26,13 +26,24 @@ export class UUITabGroupElement extends LitElement {
     `,
   ];
 
-  @queryAssignedNodes(undefined, true, 'uui-tab')
-  private slotNodes?: UUITabElement[];
+  @queryAssignedElements({
+    flatten: true,
+    selector: 'uui-tab, .uui-tab, [role=tab]',
```

Comment:
I am not a huge fan of using CSS classes as selectors. I think they should be for styling only. Maybe it will make sense to use the element name as an attribute? 

I also think we could remove the class for now. You can get far with just the role until we figure out the convention we want.

Reply:
You are right. I was contemplating the element name as an attribute, but didn't know if it would work. I like the idea, will change it up.

Reply:
Oh, something came up with this. It's because UUITab extends from `ActiveMixin` that we do this. I guess this change is a bit out of scope for this feature if we want to change the mixin?